prog.go

```diff
@@ -891,6 +891,12 @@ func findTargetInKernel(name string, progType ProgramType, attachType AttachType
 		}
 		return module, id, nil
 	}
+	// See cilium/ebpf#894. Until we can disambiguate between equally-named kernel
```

Comment:
There's been talks of including symbol addresses in Func, but I've suggested to also include source file names, otherwise users wouldn't be able to reliably specify (in ELF section names) which symbol they're interested in.

There are more than 500 (unique) ambiguous symbols in 6.1, so this is a pretty heavy limitation that should be lifted at some point.